src/App.tsx

```diff
@@ -1,223 +1,160 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useCallback,
+  useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { getTodos, addTodo, removeTodo } from './api/todos';
+import { Errors } from './components/Errors';
+import { Footer } from './components/Footer';
+import { FilterMethods } from './types/FilterMethods';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filterMethod, setFilterMethod]
+  = useState<FilterMethods>(FilterMethods.ALL);
+  const [title, setTitle] = useState('');
+  const [isAdding, setIsAdding] = useState(false);
+
+  const getTodosFromsServer = useCallback(async () => {
+    if (!user) {
+      return;
+    }
+
+    try {
+      const todosFromServer = await getTodos(user.id);
+
+      setTodos(todosFromServer);
+    } catch (Error) {
+      setErrorMessage('Loading error!');
+
+      setTimeout(() => {
+        setErrorMessage('');
+      }, 3000);
+    }
+  }, []);
```

Comment:
There's no need in useCallback as you don't pass it to child components or in any deps array.

https://beta.reactjs.org/apis/react/useCallback